specificatie/genereervariant/openapi.json

```diff
@@ -20,8 +20,8 @@
     "/volgindicaties" : {
       "get" : {
         "tags" : [ "Beheren volgindicaties" ],
-        "summary" : "Raadplegen alle actieve volgindicaties",
-        "description" : "Opvragen van alle actuele (actieve) volgindicaties van een abonnee. Volgindicaties met een einddatum in het verleden worden niet geleverd.\n",
+        "summary" : "Raadpleeg actieve volgindicaties",
+        "description" : "Vraag de actieve volgindicaties op van een abonnee. Levert geen volgindicaties met als einddatum vandaag of een einddatum in het verleden.\n",
```

Comment:
Kan toch beter :-).
overal veranderen in: Levert geen volgindicaties met einddatum vandaag of in het verleden.